core/chains/evm/txmgr/txmgr.go

```diff
@@ -155,7 +155,8 @@ func NewTxm(db *sqlx.DB, ethClient evmclient.Client, cfg Config, keyStore KeySto
 		reset:            make(chan reset),
 	}
 	if cfg.EthTxResendAfterThreshold() > 0 {
-		b.ethResender = NewEthResender(lggr, db, ethClient, keyStore, defaultResenderPollInterval, cfg)
+		orm := NewORM(db, lggr, cfg)
```

Comment:
directly pass orm into resender (insteaad of sqlx.DB)

Reply:
this will come into play later, but you would probably want to construct the ORM outside of the `cfg.EthTxResendAfterThreshold` logic since it will be used for more than just the resender